src/anjaliSDec19/Assignment2/LargestNumber.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * largest and smallest number in an array
+ */
+package anjaliSDec19.Assignment2;
+
+public class LargestNumber {
+	
+	void findLargeSmall(int[] arr)
+	{
+		int a=0;
+		for(int i=0;i<arr.length;i++)
```

Comment:
we need only single for loop to get largest number. do not use two for loop, here you are using bubble sort algorithm and find largest number, i like it, if you understood well bubble sort.
keep it up!